app/assets/stylesheets/rules.css.scss

```diff
@@ -0,0 +1,7 @@
+#rules {
+  .rules-container {
+    height: 480px;
+    overflow-y: auto;
+    border: 1px solid #111;
```

Comment:
Color literals like `#111` should only be used in variable declarations; they should be referred to via variable everywhere else.